nextflow/nextflow.config

```diff
@@ -0,0 +1,171 @@
+manifest {
+  author = "Chunjie Liu"
+  homePage = "https://github.com/chunjie-sam-liu/rmats-turbo-nf"
+  mainScript = "rmatsTurbo.nf"
+}
+
+name = "test_rmats_turbo_nf"
+workDir = "${HOME}/scratch/nf-work/${name}"
+
+params {
+  // input bam files
+  // bam_g1 = false
+  bam_g1 = "/scr1/users/liuc9/rmatsTurbo/GS689.tsv"
+  bam_g2 = "/scr1/users/liuc9/rmatsTurbo/PC3E.tsv"
+
+  // reference
+  // gtf = false
+  gtf = "/mnt/isilon/xing_lab/liuc9/refdata/star-genome-index-grch38-gencode-v36/gencode.v36.annotation.chrom.gtf"
+
+  is_single_end = false
+  readLength = 101
+  nthread = 1
+
+  out_dir = "test_out"
+  lib_type = "fr-unstranded"
+  variable_read_length = false
+  anchorLength = false
+
+  tstat = 1
+  cstat = "0.0001"
+  statoff = false
+  paired_stats  = false
+  darts_model = false
+  darts_cutoff = "0.05"
+
+  novelSS = false
+  mil = 50
+  mel = 500
+  allow_clipping = false
+  individual_counts = false
+
+
+  machine_mem_gb = 4
```

Comment:
These aren't used yet. Ideally `machine_mem_gb` could be used for `memory` for each `process` instead of `label "*_memory"`. If nexflow doesn't have something for a disk requirement then `disk_space_gb` and `use_ssd` can be removed

Reply:
Nextflow uses the `label` directive for requesting resources. It's not like the WDL. I will remove the `disk_space_db` and `use_ssd`.